modules/github-repository/branch.tf

```diff
@@ -1,7 +1,7 @@
 resource "github_branch" "branch" {
   for_each = local.branches_map
 
-  repository    = github_repository.repository.name
+  repository    = var.existing_repository == null ? github_repository.repository[0].name : data.github_repository.imported_github[0].name
```

Comment:
you use same "var.existing_repository == null ? github_repository.repository[0].name : data.github_repository.imported_github[0].name" in about 11 places, why not have local named repository_name and assign this value and use in those 11 places?

Reply:
good idea